Test/error_test.py

```diff
@@ -48,7 +48,7 @@
 Traceback (most recent call last):
         ...
 pycm.pycm_obj.pycmMatrixError: Mapping class names error
->>> cm.relabel({1:"L1",2:"L2",3:"L2"})
+>>> cm.relabel({0:"L1",1:"L2",2:"L2"})
```

Comment:
I think it couldn't be the ERROR case, apparently it might relabel classes normally.

Reply:
It's an error, mapping to same labels ;-)